pkg/controller/seed/seed_control.go

```diff
@@ -126,6 +130,33 @@ func (c *defaultControl) ReconcileSeed(obj *gardenv1beta1.Seed, key string) erro
 		seedLogger  = logger.NewSeedLogger(logger.Logger, seed.Name)
 	)
 
+	// The deletionTimestamp labels a Seed as intented to get deleted. Before deletion, it has to be ensured that no Shoots
+	// are depending on the Seed anymore. When this happens the controller will remove the Finalizers from the Seed and deletion will be triggered.
```

Comment:
See above suggestions in CloudProfile controller.

Reply:
done